.gitignore

```diff
@@ -112,3 +112,6 @@ tmp/
 dask-worker-space
 scripts/runs
 scripts/datasets
+
+# ReaLTabFormer 
+rtf_checkpoints/
```

Comment:
We don't need to add this to the gitignore

Reply:
I think we should, when I run this synthesizer locally I get this folder.

Reply:
Oh ok, agreed